_lifeofinternatpuppet/post_4.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: page
+title: "Week 4"
+author: Disha-maker
+categories:
+  - team
+  - basics
+tags:
+  - week 4
+---
+
+Hello, happy Friday everyone, 
+
+Well, this week was extremely very busy with lots going on.... 
+
+So let's get started! 
+
+The week started off my having a sync-up meeting with [Norman] where he was finding out about how my previous week was, as he was on holidays. 
+In addition to this, he also informed me that in the coming weeks, I will have to deliver a presentation around Week 5 or Week 6. 
+Stay tuned for this :) 
+
+We also decided how we are going to set up the system for verifying monthly logs for my university placement year report. 
+Lastly, [Norman] was telling me to have a dedicated page called 'Life of an Intern at Puppet' as he thinks that this will really help new interns joining the company next year and most importantly he enjoyed reading sections of my weekly updates each week. 
+
+Thank you for giving me this opportunity [Norman], much appreciated! 
+
+After talking to [Norman], I had a quick sync up with [Ciaran] and [Daiana] where I shared my thoughts on my new blog page called 'Life of an Intern at Puppet' 
```

Comment:
As we discussed offline from the PR, you're going to fix up and address the links (e.g. `[Norman], [Ciaran], [Daiana]`) at a later stage, so I'm happy for that not to hold up this PR merging for now

Reply:
Should I fix this now, can can I send a request for merging and review?

Reply:
I don't see the commit that addresses that and the links still seem to be incorrectly formatted. Did you add, commit and push the changes?

Reply:
I see your [push](https://github.com/puppetlabs/iac/pull/117/commits/3d65b1a2a57108aae680c21912cd93201f3e888f) just came in as I responded 😄

Reply:
Yes